<launch>
  <arg name="master_name" default="haptic_device"/>
  <arg name="slave_name" default="grips"/>
  
  <!-- Phantom node -->
  <node name="omni_state" pkg="omni_common" type="omni_state" output="screen"> 
    <param name="omni_name" type="string" value="$(arg master_name)"/>
    <rosparam command="load" file="$(find rate_position_controller)/config/phantom_omni.yaml" />
  </node>
  
  <!-- Rate position controller -->
  <node name="rate_position_controller" pkg="rate_position_controller" type="rate_position_controller.py" output="screen">
    <param name="master_name" type="string" value="$(arg master_name)"/>
    <param name="slave_name" type="string" value="$(arg slave_name)"/>
    <rosparam command="load" file="$(find rate_position_controller)/config/phantom_omni.yaml" />
  </node>
</launch>
